docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -55,68 +55,125 @@ If you are installing Rancher on a K3s cluster with **Raspbian Buster**, follow
 
-These CPU and memory requirements apply to each instance with RKE2 installed. Minimum recommendations are outlined here.
+Please note that a highly available setup with at least 3 nodes is required for all production usages.
 
-| Deployment Size | Clusters | Nodes     | vCPUs | RAM  |
-| --------------- | -------- | --------- | ----- | ---- |
-| Small           | Up to 5  | Up to 50  | 2     | 5 GB |
-| Medium          | Up to 15 | Up to 200 | 3     | 9 GB |
+These requirements apply hosted Kubernetes clusters such as EKS, AKS, or GKE. They do not apply to Rancher SaaS solutions such as Rancher Prime Hosted](https://www.rancher.com/products/rancher).
+
+| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
+|-----------------------------|-----------|------------|-------|-------|
+| Small                       | Up to 150 | Up to 1500 | 4     | 16 GB |
+| Medium                      | Up to 300 | Up to 3000 | 8     | 32 GB |
+| Large (*)                   | Up to 500 | Up to 5000 | 16    | 64 GB |
+
+(*): Large deployments require [additional tuning and following of best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md).
+
+### RKE
+
+Minimum CPU and memory requirements for each individual node in the [Kubernetes cluster Rancher is installed in](install-upgrade-on-a-kubernetes-cluster.md) are listed in the table below.
+
+Please note that a highly available setup with at least 3 nodes is required for all production usages.
+
+| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
+|-----------------------------|-----------|------------|-------|-------|
+| Small                       | Up to 150 | Up to 1500 | 4     | 16 GB |
+| Medium                      | Up to 300 | Up to 3000 | 8     | 32 GB |
+| Large (*)                   | Up to 500 | Up to 5000 | 16    | 64 GB |
+
+(*): Large deployments require [additional tuning and following of best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md).
+
+Refer to RKE documentation for more detailed information on [RKE general requirements](https://rke.docs.rancher.com/os).
 
 ### Docker
 
-These CPU and memory requirements apply to a host with a [single-node](rancher-on-a-single-node-with-docker.md) installation of Rancher.
+Minimum CPU and memory requirements for a [single node installation of Rancher](rancher-on-a-single-node-with-docker.md) are listed in the table below.
+
+Please note that a Docker installation is only suitable for development or testing purposes and is not meant to be used in production environments.
 
-| Deployment Size | Clusters | Nodes     | vCPUs | RAM  |
-| --------------- | -------- | --------- | ----- | ---- |
-| Small           | Up to 5  | Up to 50  | 1     | 4 GB |
-| Medium          | Up to 15 | Up to 200 | 2     | 8 GB |
+| Managed Infrastructure Size | Clusters | Nodes     | vCPUs | RAM  |
```

Comment:
Instead of "Up to" in each cell, revise to have the header row state "Maximum # of Clusters" & "Maximum # of Nodes"

Reply:
Addressed as of 548e4de67ae, resolving.